CHANGELOG.rst

```diff
@@ -1,15 +1,31 @@
 Changelog
 =========
 
-1.3.0a2 (2023-11-23)
+1.3.0a3 (2023-12-07)
 --------------------
 
 Features
 
 * Add ``ProfileMapping`` for Vertica by @perttus in #540 and #688
-* Add ``ProfileMapping`` for Snowflake encrypted private key path by @ivanstillfront in #608
 * Add support for Snowflake encrypted private key environment variable by @DanMawdsleyBA in #649
-* Add ``DbtDocsGCSOperator`` for uploading dbt docs to GCS by @jbandoro in #616
```

Comment:
Should we keep these two that were removed here to show they are still in the 1.3 alpha release

Reply:
Great catch! It was a mistake, just fixed, @jbandoro - thank you!